retrace/glws_egl_xlib.cpp

```diff
@@ -129,6 +129,13 @@ class EglDrawable : public Drawable
         eglWaitNative(EGL_CORE_NATIVE_ENGINE);
     }
 
+    void
+    setDamageRegion(int *rects, int nrects) override {
+        EGLint value;
+        eglQuerySurface(eglDisplay, surface, EGL_BUFFER_AGE_KHR, &value);
```

Comment:
You should check here that the respective EGL extension is supported, and do nothing if not.